modules/ROOT/partials/misc/inline-formatting-of-list-bullets.adoc

```diff
@@ -0,0 +1,22 @@
+== Inline formatting.
```

Comment:
I think we will need to include `include::partial$misc/admon-requires-6.2v.adoc[]` as this behaviour will only work as stated from v6.2 onwards.

Reply:
`include` included.